src/models/hooks/activityReport.js

```diff
@@ -230,7 +230,7 @@ const determineObjectiveStatus = async (activityReportId, sequelize, isUnlocked)
       // Get latest report by end date.
       const latestAR = relevantARs.reduce((r, a) => {
         if (r && r.endDate) {
-          return r.endDate > a.endDate ? r : a;
+          return new Date(r.endDate) > new Date(a.endDate) ? r : a;
```

Comment:
is this happening anywhere else?

Reply:
Looks like maybe a few:
./src/services/recipient.js:204:        endDate: report.endDate > a.endDate ? report.endDate : a.endDate,
./src/services/recipient.js:253:    a.endDate === b.endDate ? a.id < b.id
./src/services/recipient.js:254:      : a.endDate < b.endDate) ? 1 : -1));
./src/widgets/totalHrsAndRecipientGraph.js:129:      if (!arDates.find((cache) => cache.id === r.id && cache.date === r.startDate)) {

Reply:
This was the test I ran.

// Check === with Dates
const dateTestE = '01/05/2023';
const dateELess = '01/05/2023';
const checkE = new Date(dateTestE) === new Date(dateELess);
console.log('ResultE: ', checkE);

// Check === with Strings
const dateTestF = '01/05/2023';
const dateFLess = '01/05/2023';
const checkF = dateTestF === dateFLess;
console.log('ResultF: ', checkF);